site/content/installation/installing-ngf/helm.md

```diff
@@ -116,6 +124,14 @@ helm install ngf oci://ghcr.io/nginxinc/charts/nginx-gateway-fabric --create-nam
 
 {{<note>}}Requires the Gateway APIs installed from the experimental channel.{{</note>}}
 
+#### Examples
+
+You can find several examples of configuration options of the `values.yaml` file in the [helm examples](https://github.com/nginxinc/nginx-gateway-fabric/tree/v1.3.0/examples/helm) directory.
+
+### Access NGINX Gateway Fabric
+
+{{<include "installation/expose-nginx-gateway-fabric.md" >}}
```

Comment:
Does this doc stand by itself and get included here?

Reply:
I think it makes sense to just include it